platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheBuildTreeIO.kt

```diff
@@ -67,23 +71,28 @@ interface ConfigurationCacheBuildTreeIO : ConfigurationCacheOperationIO {
      * @param profile the unique name associated with the output stream for debugging space usage issues
      */
     fun writeContextFor(
+        name: String,
```

Comment:
Really hard to reason about state involving multiple contexts without having an easy way to tell them apart